Source/OxyPlot/Axes/Axis.cs

```diff
@@ -1685,22 +1685,13 @@ protected double CalculateActualInterval(double availableSize, double maxInterva
         /// <returns>The minor interval.</returns>
         protected double CalculateMinorInterval(double majorInterval)
```

Comment:
This method should be covered by unit tests, I think. The new implementation is not totally clear to me.. :-)
I would try test cases: 1, 2, 5, 10, 25 etc.